tools/topology/sof-icl-rt711-rt1308-rt715-hdmi.m4

```diff
@@ -190,7 +191,7 @@ DAI_CONFIG(ALH, 3, 1, SDW0-Capture)
 DAI_CONFIG(ALH, 0x102, 2, SDW1-Playback)
 
 #ALH SDW2 Pin2 (ID: 3)
-DAI_CONFIG(ALH, 0x202, 3, SDW2-Playback)
+ifelse(PLATFORM, `icl', DAI_CONFIG(ALH, 0x202, 3, SDW2-Playback),`')
```

Comment:
put the if else above the SDW2, that way all references to SDW2 and ALH0x202 are gone from the .conf